build/db/entities/submission.js

```diff
@@ -71,7 +71,7 @@ var Submission = /** @class */ (function() {
   __decorate(
     [
       typeorm_1.Column({
-        type: "smallint",
+        type: "integer",
```

Comment:
Why to you want to increase it to an int? Smallints' value can go up to 32,767.
Moreover, if you really want to change it, you should move this change to `./lib/db/entities/submission.ts`. This is the built file, so changing things here would have effects :)

Reply:
Oh, thanks you checked carefully, I could have break the thing! 
The reason why I wanted to increase it to int was to avoid un-explained failure, in case the user was sick more than a month ago and - _god knows why /even if not relevant_ -  wants to mention it, it should not fail. Just like it is very unlikely that someone is confined with more than 32 people but if it happens, it won't fail. An other possibility could be to check when posting the report and if greater than smallints' limit, set to null? 

I reverted that change though :)